sdk/python/jobs/pipelines/1i_pipeline_with_spark_nodes/components/spark_kmeans.yml

```diff
@@ -0,0 +1,34 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
```

Comment:
a more usual structure is:
1i/
  pipeline.yml
  component.yml
  src/
    kmeans_example.py

Reply:
offline synced